charts/pomerium/templates/_helpers.tpl

```diff
@@ -401,7 +401,7 @@ certificate_file: "/pomerium/cert.pem"
 certificate_key_file: "/pomerium/privkey.pem"
 certificate_authority_file: "/pomerium/ca.pem"
 {{- end }}
-authenticate_service_url: {{ default (printf "https://authenticate.%s" .Values.config.rootDomain ) .Values.proxy.authenticateServiceUrl }}
+authenticate_service_url: {{ default (printf "https://%s.%s" (.Values.ingress.authenticate.name | default "authenticate") .Values.config.rootDomain ) .Values.proxy.authenticateServiceUrl }}
```

Comment:
I think you can use the `pomerium.authenticate.hostname` template here too

Reply:
done